sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/models/package-info.java

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
This is not true, right?

Reply:
Do you mean this line? `// Code generated by Microsoft (R) AutoRest Code Generator.` 
This file is generated by autorest.